### DISCUSSION TITLE
Self-made chrome alum gelatin coated slides (Protocol)

### DISCUSSION THREAD

**Discussion by sfritzs** [IBEXImagingCommunity/ibex_imaging_knowledge_base#133] [Show and tell]
Hi everyone, 

since a few of you (including our lab) do not seem to be able to order the chrom alum gelatine that is mentioned in the fantastic IBEX paper from outside the US, we tried out different recipes to recreate the commercial solution.

This is what works for us:

1. 	Heat up 200ml ddH2O to 50°C on hot plate using a magnetic stirrer
2. 	Add 1g gelatin gradually until completely dissolved to prevent formation of clumps. Solution has to be clear.
3. 	Let solution cool down to room temperature and add 0.1g Chrome potassium sulfate until completely dissolved.
        Either proceed immediately or store the solution for a few days @ 4°C. If stored, let the solution warm-up to room temperature 
        before use.
4.	Pour the solution in a staining dish preheated at 60 ºC.
5.	Incubate to-be-treated slides in 70% Ethanol for 10 minutes to clean them.
6.	Rinse in distilled water, dip at least 3 times to remove alcohol.
7. 	Dip the slides 5 times in the solution, 10 seconds each.
8. 	Leave the slides to drain for a few minutes onto a filter paper. 
9. 	Dry the treated slides in an oven at 37 ºC over night.
10. 	Store the slides in a dry box and protect them from dust.

Materials:
- Chromium(III) potassium sulphate dodecahydrate ([#3535.3 Carl Roth](https://www.carlroth.com/de/de/von-a-bis-z/chrom%28iii%29-kaliumsulfat-dodecahydrat/p/3535.3))
- Gelatine from porcine skin ([#48722 Sigma-Aldrich](https://www.sigmaaldrich.com/DE/de/product/sial/48722))

**Comment by radtkea** [comment]
Wow sfritzs!

Thank you so much for sharing your protocol for making chrome alum-gelatin adhesive! It will definitely help users around the globe! 

Would you be able to write it up as a protocol for [protocols.io](https://www.protocols.io/)?

The advantage is it is linked to your ORCID and we can link to your protocol [here](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/protocols.html) and acknowledge you on our contribute [page](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/the_who.html).

We need to do a new release on Zenodo. When we do that, you will be named on the [site](https://zenodo.org/records/7693279) and can add your contribution to your CV. From our homepage: “Open science is obviously something you support and practice. Don’t forget to highlight this in your biosketch or c.v. If using the NIH biosketch format, we recommend listing the knowledge-base under the “Contributions to Science” section. If using a generic c.v. format, we recommend listing the knowledge-base under a “Datasets” section.”

Thank you again! You are terrific:)
A

**Comment by sfritzs** [comment]
Hi Andrea,

you can find the protocol here: https://dx.doi.org/10.17504/protocols.io.3byl49kkogo5/v1

Hope this works for everyone! 

All the best, 
Sonja

**Reply by radtkea** [reply]
Thank you so much Sonja! This is terrific!!! I will work with @zivy to post on the IBEX Imaging Community and share in our protocols.io workspace.

**Comment by zivy** [comment]
Hi @radtkea, just requested your review on [this pull request](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/pull/136) which adds the protocol to the knowlege-base. Once you approve will merge it in.

**Comment by zivy** [comment]
Hello @sfritzs,

Thanks for contributing to the knowledge-base. 

The protocol is now listed on the [protocols page](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/protocols.html) and you are listed on the [contributors page](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/the_who.html) (in the next [zenodo knowledge-base release](https://zenodo.org/records/7693279) you will appear on the author byline).
